### ISSUE TITLE
[Bug]: Low perfromance of the TF quantized model.

### ISSUE THREAD

**Issue by alexsu52** [openvinotoolkit/openvino#19019]
### OpenVINO Version

Master branch

### Operating System

Ubuntu 20.04 (LTS)

### Device used for inference

CPU

### Framework

Keras (TensorFlow 2)

### Model used

Mobilenet v2

### Issue description

Converted TF qunatized mobilent v2 is not performed comparing with OpenVINO 2023.0.1.

[tensorflow_model.zip](https://github.com/openvinotoolkit/openvino/files/12279101/tensorflow_model.zip)

### Step-by-step reproduction

```
tf_quantized_model = tf.saved_model.load("path_to_model") 
ov_quantized_model = mo.convert_model(tf_quantized_model)
```

Use benchmark_app to benchmark FP32 and INT8 models. 

Also you can use NNCF example to reproduce this issue using OpenVINO from the master branch https://github.com/openvinotoolkit/nncf/tree/develop/examples/post_training_quantization/tensorflow/mobilenet_v2

### Issue submission checklist

- [X] I report the issue. It's not a question
- [X] I checked the problem with the documentation, FAQ, open issues, Stack Overflow, etc., and have not found the solution
- [X] There is reproducer code and related data files such as images, videos, models, etc.

**Comment by andrei-kochin** [comment]
Ref: CVS-117250

**Comment by alexsu52** [comment]
@rkazants, thanks for reply! I validated the release/2023/1 branch and the issue is still here. Could you try to reproduce on the release branch.

**Comment by andrei-kochin** [comment]
@alexsu52 issue was fixed in scope of #19735
